src/components/staking/PageValidator.vue

```diff
@@ -19,138 +24,128 @@
-                  <div class="page-profile__title">{{ validator.description.moniker }}</div>
-                </div>
-                <Bech32 :address="validator.operator_address" />
-              </div>
-            </div>
-
-            <div class="page-profile__header__actions">
-              <TmBtn
-                id="delegation-btn"
-                :disabled="!connected"
-                :value="connected ? 'Delegate' : 'Connecting...'"
-                color="primary"
-                @click.native="onDelegation"
-              />
-              <TmBtn
-                id="undelegation-btn"
-                :disabled="!connected"
-                :value="connected ? 'Undelegate' : 'Connecting...'"
-                color="secondary"
-                @click.native="onUndelegation"
-              />
+          <div class="validator-info">
+            <h3 class="li-validator-name">
+              {{ validator.description.moniker }}
+            </h3>
+            <div v-if="myDelegation">
+              <h4>{{ myDelegation }}</h4>
+              <h5 v-if="rewards">
+                {{ rewards ? `+` + shortDecimals(atoms(rewards)) : `--` }}
+              </h5>
             </div>
           </div>
-        </div>
+        </td>
+      </tr>
 
-        <div class="row">
-          <div class="row row-unjustified">
-            <dl class="info_dl colored_dl">
-              <dt>My Delegation</dt>
-              <dd>{{ myDelegation }}</dd>
-            </dl>
-            <dl class="info_dl colored_dl">
-              <dt>My Rewards</dt>
-              <dd v-if="rewards > 0">
-                {{ rewards | atoms | shortDecimals }}
-                {{ bondDenom | viewDenom }}
-              </dd>
-              <dd v-else>--</dd>
-            </dl>
-            <dl class="info_dl colored_dl">
-              <dt>Expected Returns</dt>
-              <dd>{{ percent(returns) }}</dd>
-            </dl>
-          </div>
+      <div class="button-container">
+        <TmBtn id="delegation-btn" value="Stake" @click.native="onDelegation" />
+        <TmBtn
+          id="undelegation-btn"
+          :disabled="!myDelegation"
+          value="Unstake"
+          type="secondary"
+          @click.native="onUndelegation"
+        />
+      </div>
 
-          <div class="row row-unjustified">
-            <dl class="info_dl colored_dl">
-              <dt>Voting Power</dt>
-              <dd id="page-profile__power">{{ percent(powerRatio) }}</dd>
-            </dl>
-            <dl class="info_dl colored_dl">
-              <dt>Uptime</dt>
-              <dd id="page-profile__uptime">{{ uptime }}</dd>
-            </dl>
-            <dl class="info_dl colored_dl">
-              <dt>Commission</dt>
-              <dd id="page-profile__commission">{{ percent(validator.commission.rate) }}</dd>
-            </dl>
-          </div>
-        </div>
+      <div class="row">
+        <dl class="info_dl">
```

Comment:
2 comments:
1. Could call it `info_param`, we can see it's applied to a `<dl>`.
2. We're misusing the `<dl>`. It should just be one `<dl>…</dl>` with multiple children `<dt>…</dt><dd>…</dd>` pairs.

Reply:
@jbibla